op-challenger/game/fault/trace/cannon/provider.go

```diff
@@ -194,3 +213,26 @@ func (p *CannonTraceProvider) loadProof(ctx context.Context, i uint64) (*proofDa
 	}
 	return &proof, nil
 }
+
+type diskStateCacheObj struct {
+	Step uint64 `json:"step"`
+}
+
+func ReadLastStep(dir string) (uint64, error) {
```

Comment:
Would be nice to have some comments on what these functions do / how the last step value relates to what snapshots exists.